General tidy up! (#56)
ietf-rats/draft-deshpande-multi-verifier: draft-deshpande-rats-multi-verifier.md

Nonetheless, such benefits depend on strong implementation practices.

- Minimization: Attesters should only generate Evidence that is strictly necessary for the appraisal policy. Verifiers should only request necessary claims.

> - Confidentiality: Encryption should be used so that sensitive Evidence can be accessed only by the Verifier that is legitimately intended to receive it, and by no unauthorized parties (including other Verifiers in the hierarchy or cascade). This is crucial in multi-tenant environments.

Reviewer: ```suggestion
- Confidentiality: Evidence containing sensitive information should be encrypted so that it can only be accessed by the intended Verifier and not by any unauthorised parties (including other Verifiers in the hierarchy or cascade). This is crucial in multi-tenant environments.
```

> - Confidentiality: Encryption should be used so that sensitive Evidence can be accessed only by the Verifier that is legitimately intended to receive it, and by no unauthorized parties (including other Verifiers in the hierarchy or cascade). This is crucial in multi-tenant environments.

Reviewer 2: - Confidentiality: Encryption should be used so that sensitive Evidence can be accessed only by the Verifier that is legitimately intended to receive it, and by no unauthorized parties (including other Verifiers in the hierarchy, cascade or hybrid mode). This is crucial in multi-tenant environments.
